app/views/self_service/offices/_form.html.erb

```diff
@@ -70,6 +70,14 @@
     <%= f.text_field :telephone_number, class: 't-telephone-number' %>
   <% end %>
 
+  <%= f.form_row :website do %>
+    <%= f.errors_for :website %>
+    <%= f.label :website,
+                Office.human_attribute_name(:website),
+                class: 'form__label-heading' %>
+    <%= f.text_field :website, class: 't-website' %>
```

Comment:
@jriga just to give some context, the `t-` prefix is a convention for classes just in place for the purpose of testing.

Additionally, since we use SitePrism, any `t-` prefix classes also would be typically accompanied by something in one of the files in `features/support/ui/pages` referring to it.